tests/infrastructure/infrastructure.go

```diff
@@ -25,6 +25,10 @@ import (
 	"kubevirt.io/kubevirt/tests/decorators"
 )
 
-func DescribeInfra(text string, args ...interface{}) bool {
+func DescribeSerialInfra(text string, args ...interface{}) bool {
 	return Describe("[Serial][sig-compute]Infrastructure "+text, Serial, decorators.SigCompute, args)
```

Comment:
`[Serial]` in the description text should not be required anymore. The `Serial` decorator is sufficient.

Reply:
True! But we need to wait for https://github.com/kubevirt/kubevirt/pull/13321 to be merged